### DISCUSSION TITLE
[Vuetify] [BREAKING] 'app' has been removed, use '<v-app-bar app>' instead.

### DISCUSSION THREAD

**Discussion by santosh** [vuetifyjs/vuetify#13028] [Q&A] [answered]
I am following the Vuetify playlist from The Net Ninja. This is my [current state of the application](https://gitlab.com/frontend-incubation/todo-ninja/-/tree/fc0a607f4b8a05c8f1af9bad2ff19ccd6b7a4060). 

I know the tutorial is quite outdated with the today's standard and I'm transforming to the latest semantics wherever possible. But this blocker is not going and I'm not sure how will it go. 

This is the complete error:

![image](https://user-images.githubusercontent.com/1515991/106303066-b8755d00-627f-11eb-8f77-28c5f3c73f0b.png)

Any idea guys?

**Comment by jacekkarczmarczyk** [comment, marked as answer]
Use `<v-app-bar app>` instead of `<v-toolbar app>`

**Reply by santosh** [reply]
Yes, that fixes the problem. 

So what is the difference?

**Reply by MajesticPotatoe** [reply]
in v2, v-toolbar was split into 2 different components to scope functionality. All of the v1.5 functionality got put into v-app-bar and added some additional functionality like scrolling techniques, and v-toolbar became a generic toolbar and lost its `app` functions.
